src/components/patterns/horizontal-card.tsx

```diff
@@ -49,54 +47,72 @@ const HorizontalCard = ({
     linkAttributes["aria-labelledby"] = headerId
     delete linkAttributes["aria-label"]
   }
+  let bgColor = "bg-fog-light text-black-true"
+  let btnColor = "bg-cardinal-red text-white border-cardinal-red hocus:bg-black-true hocus:text-white"
+  let linkColor = ""
 
-  return (
-    <div className="relative" {...props} ref={ref}>
-      {fullWidth && (
-        <div className="absolute left-0 top-0 z-[-10] ml-[calc(-50vw+50%)] h-full w-screen">
-          <div className="relative h-full w-full bg-black-true" {...props}>
-            <CardSprinkles position={backgroundSprinkles} />
-          </div>
-        </div>
-      )}
-
-      {!fullWidth && (
-        <FullScreenBackground compareRef={ref} className="relative h-full w-full bg-black-true">
-          <CardSprinkles position={backgroundSprinkles} />
-        </FullScreenBackground>
-      )}
+  switch (cardBgColor) {
+    case "cardinal_red":
+      bgColor = "bg-cardinal-red text-white"
+      btnColor = "bg-white text-cardinal-red hocus:bg-black-true hocus:text-white border-white"
+      linkColor = "[&_a]:text-white hocus:[&_a]:text-black-true"
+      break
+  }
```

Comment:
`clsx` would be a good use here:
```
clsx({
  "bg-red": cardBgColor === "cardinal_red",
  "bg-fog": cardBgColor !== "cardinal_red",
})
```

That way when more colors are added, it'll be simple to update.